### ISSUE TITLE
Add a 'auto' imagej like option

### ISSUE THREAD

**Issue by payno** [silx-kit/silx#4123]
IT would be interesting to have (yet) another option for defining the colormap min/max values.

Closer to what imageJ is doing:

![image](https://github.com/silx-kit/silx/assets/12907796/12018315-67dd-4aa2-8621-ecd764a33462)

note: Suggested by Elodie

**Comment by t20100** [comment]
Can you provide a link to the relevant doc, the screen shot is not readable

**Comment by payno** [comment]
Yes sorry. Here is the link: https://imagej.net/ij/docs/guide/146-28.html

**Comment by payno** [comment]
Indeed this is the 'percentile' colormap but with different values. We can add a 'saturation' spin box to the colormap to have this 'mode'

**Comment by payno** [comment]
done in https://github.com/silx-kit/silx/pull/4379
